src/components/Input.js

```diff
@@ -18,23 +19,40 @@ const Input = ({ name, ...props }) => {
           input: { value },
           id,
           label,
+          sublabel,
           hidden,
           icon,
           iconPosition,
           colspan,
           inputProps,
+          sublabelProps,
         } = fieldProps;
 
         return (
           <SuiForm.Field
             width={colspan}
-            style={{ visibility: hidden ? 'hidden' : 'visible' }}
+            // ! TODO: add animation so the form doesn't jump when showing and hiding component
+            style={{ display: hidden ? 'none' : 'initial' }}
           >
             {label && (
               <label htmlFor={id || name} style={{ whiteSpace: 'pre' }}>
                 {label}
               </label>
             )}
+            {sublabel && (
+              <SuiHeader
+                {...sublabelProps}
```

Comment:
Give me an example of using `sublabelProps`

Reply:
Please refer to the comment above.